doc/source/how-to/vulnerabilities.rst

```diff
@@ -0,0 +1,146 @@
+
+Vulnerability discovery and tracking
+-------------------------------------
+
+Leverage available security tools, such as dependency scanners or static
+analyzers (such as PyUp, Safety, or Bandit), to automatically detect and
+remediate security vulnerabilities in Python packages and dependencies.
+
+The PyAnsys ecosystem has implemented automated mechanisms to track and
+report vulnerabilities in the codebase. These tools are intended to be integrated
+into the CI/CD pipeline of the repositories.
+
+The PyAnsys Core team has come up with a GitHub Action that can be used to
+automatically check for vulnerabilities in the codebase. This action is based on
+the following tools:
+
+- `Safety`_ - checks installed dependencies for known security vulnerabilities
+- `Bandit`_ - a tool designed to find common security issues in Python code
+
+Safety addresses external dependencies, while Bandit focuses on the codebase itself.
+Both tools have been integrated into the action ``ansys/actions/check-vulnerabilities``.
+
+.. note::
+
+  The tools selected for the action are not definitive. The action can be modified to include
+  other tools or to use different versions of the tools in the future.
+
+For third-party packages, the PyAnsys Core team has listed a set of excluded advisories by which
+the action will not fail. This is done to avoid false positives and to ensure that the action
+does not block the CI/CD pipeline unnecessarily. You can find the list of excluded advisories
+in `the action's documentation`_.
+
+For potential vulnerabilities in the codebase, repositories can configure `Bandit`_ to ignore specific
+advisories. This can be due to the fact that the code is not yet ready to be fixed or that the advisory is not relevant
+to the codebase. However, it is important to note that ignoring advisories should be done with caution
+and developers should be aware of the potential risks involved. Furthermore, repository maintainers should
+document the reasons for ignoring advisories and ensure that they are regularly reviewed to determine if they
+can be addressed.
+
+.. note::
+
+  An example on how to document the ignored advisories can be found in the `PyACP Security Considerations`_
+  documentation page. This should be taken as a reference for documenting ignored advisories in other repositories.
+
+.. warning::
+
+  Testing the action locally before enabling it in the CI/CD pipeline is recommended. Information on how
+  to do this can be found in `the action's documentation`_.
+
+Vulnerability remediation and reporting
+----------------------------------------
+
+When a vulnerability is detected, the action will fail and report the vulnerabilities found in the codebase.
+These vulnerabilities will be reported as draft Security advisories in the repository's
+Security tab. Maintainers are then responsible for reviewing the advisories and deciding whether to address
```

Comment:
```suggestion
**Security** tab. Maintainers are then responsible for reviewing the advisories and deciding whether to address
```
Tab or page? UI elements should be formatted in bold.

Reply:
Tab and page 😄 - on the UI it's a tab, but it redirects you to a different page